components/server/src/authorization/relationship-updater.ts

```diff
@@ -132,39 +129,22 @@ export class RelationshipUpdater {
         return orgs;
     }
 
-    // max number of relationships that can be created per request by SpiceDB
-    static maxRelationshipUpdates = 1000;
-    // how many relationships are created per workspace
-    static relationshipsPerWorkspace = 2;
-    static workspacesPageSize =
-        RelationshipUpdater.maxRelationshipUpdates / RelationshipUpdater.relationshipsPerWorkspace;
-
     private async updateWorkspaces(user: User): Promise<void> {
-        let offset = 0;
-        let currentPageSize = 0;
-
-        do {
-            // Because the total amount of workspaces per user can be very large, we want to paginate through them.
-            const workspaces = await this.workspaceDB.findAllWorkspaces(
-                offset,
-                RelationshipUpdater.workspacesPageSize,
-                "creationTime",
-                "ASC",
-                {
-                    ownerId: user.id,
-                    type: "regular",
-                },
-            );
-
-            currentPageSize = workspaces.rows.length;
+        const workspaces = await this.workspaceDB.find({
+            userId: user.id,
+            includeHeadless: false,
+            includeWithoutProject: true,
+            limit: 500, // The largest amount of workspaces is 189 today (2023-08-24)
```

Comment:
I checked the prod DB the largest amount is 189, so we don't need to build/do any paging here